lib/backend/syncersv1/updateprocessors/clusterinfoprocessor.go

```diff
@@ -27,8 +27,20 @@ func NewClusterInfoUpdateProcessor() watchersyncer.SyncerUpdateProcessor {
 	return NewConfigUpdateProcessor(
 		reflect.TypeOf(apiv2.ClusterInformationSpec{}),
 		DisallowAnnotations,
-		func(node, name string) model.Key { return model.HostConfigKey{Hostname: node, Name: name} },
-		func(name string) model.Key { return model.GlobalConfigKey{Name: name} },
+		func(node, name string) model.Key {
+			if name == "DatastoreReady" {
+				return model.HostReadyFlagKey{
```

Comment:
Do you think it's worth having the Host ready flag? or should we just handle this returning nil?

Reply:
Think I prefer nil, let's do that.